packages/suite/src/support/messages.ts

```diff
@@ -7618,7 +7633,8 @@ export default defineMessages({
     },
     TR_COINJOIN_ANONYMITY_LEVEL_SETUP_DESCRIPTION: {
         id: 'TR_COINJOIN_ANONYMITY_LEVEL_SETUP_DESCRIPTION',
-        defaultMessage: 'Shows the number of people with whom your resources are indistinguishable',
+        defaultMessage:
```

Comment:
If you change defaultMessage for existing key, it will be reverted by the next crowdin sync. Only way to change existing messages is via crowdin.

Reply:
I updated all messages in crowdin as well.